.github/workflows/doc.yml

```diff
@@ -0,0 +1,53 @@
+name: Documentation
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-latest ]
+        python-version: [ '3.8' ]
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Cache emodb
+      uses: actions/cache@v3
+      with:
+        path: emodb-src
+        key: emodb-ubuntu
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v4
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Ubuntu - install libsndfile
+      run: |
+        sudo apt-get update
+        sudo apt-get install --no-install-recommends --yes libsndfile1
```

Comment:
do we need libsndfile for the documentation?

Reply:
and if so, can't we install it together with `graphviz`?

Reply:
I have removed `libsndfile1` as we do indeed not need it and combined the two sections.